.github/workflows/php-test-plugins.yml

```
@@ -0,0 +1,78 @@
+name: Unit Testing for Plugins
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    # Only run if PHP-related files changed.
+    paths:
+      - '.github/workflows/php-test-plugins.yml'
+      - 'plugins/**.php'
+      - '.wp-env.json'
+      - '**/package.json'
+      - 'package-lock.json'
+      - 'phpunit.xml.dist'
+      - 'tests/multisite.xml'
+      - 'composer.json'
+      - 'composer.lock'
+  pull_request:
+    branches:
+      - trunk
+      - 'release/**'
+      - 'feature/**'
+    # Only run if PHP-related files changed.
+    paths:
+      - '.github/workflows/php-test-plugins.yml'
+      - 'plugins/**.php'
+      - '.wp-env.json'
+      - '**/package.json'
+      - 'package-lock.json'
+      - 'phpunit.xml.dist'
+      - 'tests/multisite.xml'
+      - 'composer.json'
+      - 'composer.lock'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  php-test-plugins:
+    name: "PHP ${{ matrix.php }} / WP ${{ matrix.wp }}"
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['8.2', '8.1', '8.0', '7.4', '7.3', '7.2', '7.1', '7.0']
+        wp: [ 'latest' ]
+        include:
+          - php: '7.4'
+            wp: '6.3'
+          - php: '8.3'
+            wp: 'trunk'
+    env:
+      WP_ENV_PHP_VERSION: ${{ matrix.php }}
+      WP_ENV_CORE: ${{ matrix.wp == 'trunk' && 'WordPress/WordPress' || format( 'https://wordpress.org/wordpress-{0}.zip', matrix.wp ) }}
```

Question: Why not use ternary expression here? Is it allowed?

```suggestion
      WP_ENV_CORE: ${{ matrix.wp == 'trunk' ? 'WordPress/WordPress' : format( 'https://wordpress.org/wordpress-{0}.zip', matrix.wp ) }}
```

---

This was added as part of #544. If we need to update this, then we also have to update similar settings for other workflows (php-test.yml and php-test-standalone-plugins.yml).

Shall we open a follow-up PR to make these changes?

---

Follow-up PR is fine. Just seems hard to read right now.

---

@westonruter, GitHub workflow does not allow `${{ x ? 'ifTrue' : 'ifFalse' }}` ternary operator. Instead, it uses `${{ x && 'ifTrue' || 'ifFalse' }}`. Therefore, we don't need to open a follow-up PR for this work.